src/main/java/uk/gov/hmcts/dm/service/FileContentVerifier.java

```diff
@@ -55,10 +58,23 @@ public boolean verifyContentType(MultipartFile multipartFile) {
             }
             String detected = tika.detect(tikaInputStream, metadata);
             if (mimeTypeList.stream().noneMatch(m -> m.equalsIgnoreCase(detected))) {
-                log.error(
-                    String.format("Warning. The mime-type of uploaded file is not white-listed: %s", detected));
+                log.error("Warning. The mime-type of uploaded file is not white-listed: {}", detected);
                 return false;
             }
+
+            if (PDF_MIME.equalsIgnoreCase(detected)) {
+                try {
+                    PDDocument document = PDDocument.load(tikaInputStream.getFile());
+                    if (document.isEncrypted()) {
+                        log.error("Warning. PDF file is encrypted");
```

Comment:
Can we add in the file name to error log?

Reply:
https://github.com/hmcts/document-management-store-app/pull/1488/commits/7d4ae7546e0e2b536a533d0d7833173f7203f330